hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -5524,6 +5524,11 @@ public int getBlockCapacity() {
     return blockManager.getCapacity();
   }
 
+  @Metric({"PendingSPSPaths", "The number of paths to be processed by storage policy satisfier"})
+  public int getPendingSPSPaths() {
```

Comment:
Do you not plan to add this method to ``FSNamesystemMBean``?
if no specific reason, we should add it there, this is quite specific to FSN